### PR TITLE
Writing language/accent to backend with profile form

### PR DESCRIPTION
**Pull request by performance-fleece**
# Description

talentprofile is sending language/accent selections to the backend for profile creation

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: voicer-client/src/components/TalentProfile.js [hunk 1]
```diff
@@ -15,14 +98,86 @@ class TalentProfile extends React.Component {
...
+    if (languages === null) {
+      languages = [];
+    } else {
+      this.setState({ languages });
+    }
+  };
+
+  destructureLanguages = languages => {
+    const newArray = languages.map(lang => lang.value);
+    console.log(newArray);
```
Comment (HaBuDeSu) [inline voicer-client/src/components/TalentProfile.js]: remvoe console.log

File: voicer-client/src/components/TalentProfile.js [hunk 2]
```diff
@@ -1,12 +1,95 @@
...
+      accents: [],
+      biography: ''
+    };
+  }
+
+  componentDidMount() {
+    this.props.getLanguages().then(this.modifyLanguage);
+    this.props.getAccents().then(this.modifyAccents);
+  }
+
```
Comment (HaBuDeSu) [inline voicer-client/src/components/TalentProfile.js]: might want to add a comment here about how modifyLanguage is working with the Redux store.

File: voicer-client/src/components/TalentProfile.js [hunk 3]
```diff
@@ -15,14 +98,86 @@ class TalentProfile extends React.Component {
...
+  handleAccentChange = accents => {
+    this.setState({ accents });
+  };
+
   handleSubmit = event => {
     event.preventDefault();
+    this.submitTalentLanguages(this.state.languages);
+    this.submitTalentAccents(this.state.accents);
+    // this.destructureLanguages(this.state.languages);
```
Comment (HaBuDeSu) [inline voicer-client/src/components/TalentProfile.js]: remove commented code

File: voicer-client/src/components/TalentProfile.js [hunk 4]
```diff
@@ -15,14 +98,86 @@ class TalentProfile extends React.Component {
...
     console.log('Profile Saved');
```
Comment (HaBuDeSu) [inline voicer-client/src/components/TalentProfile.js]: remove console.log

File: voicer-client/src/components/TalentProfile.js [hunk 5]
```diff
@@ -15,14 +98,86 @@ class TalentProfile extends React.Component {
...
   };
 
+  componentDidMount() {
```
Comment (HaBuDeSu) [inline voicer-client/src/components/TalentProfile.js]: from line 138 through 177 looks like duplicate?

File: voicer-client/src/components/TalentProfile.js (hunk 1 above)
Comment (performance-fleece) [reply voicer-client/src/components/TalentProfile.js]: Removed

File: voicer-client/src/components/TalentProfile.js (hunk 2 above)
Comment (performance-fleece) [reply voicer-client/src/components/TalentProfile.js]: added Comment explaining CDM process and modifyLanguage

File: voicer-client/src/components/TalentProfile.js (hunk 3 above)
Comment (performance-fleece) [reply voicer-client/src/components/TalentProfile.js]: removed commented code

File: voicer-client/src/components/TalentProfile.js (hunk 4 above)
Comment (performance-fleece) [reply voicer-client/src/components/TalentProfile.js]: removed console.log

File: voicer-client/src/components/TalentProfile.js (hunk 5 above)
Comment (performance-fleece) [reply voicer-client/src/components/TalentProfile.js]: removed duplicate code

**Comment by performance-fleece** [comment]
Ready for review after resolving issues raised in comments.

**Pull request merged**